src/types.tsx

```diff
@@ -2,3 +2,10 @@ export type Attempt = {
   x: number
   y: number
 }
+
+export type Segment = {
+  name: string
+  pbTime: number
+  goldTime: number
+  SegmentHistory: Attempt[] | null
```

Comment:
segmentHistory ?

Reply:
This is 100% fine. Something I often consider is attempting to name arrays as the plural of whatever they are. History _alludes_ to a list of events, and an alternative like `historicalSegments`/`pastSegments` may be less clear/verbose, but /shrug.

Reply:
Going to go go with 'attemptHistory' as it's effectively the same information as the whole run attempt history we store in App, just for each segment of the run instead